test/libsolidity/SMTCheckerTest.cpp

```diff
@@ -65,3 +73,23 @@ TestCase::TestResult SMTCheckerTest::run(ostream& _stream, string const& _linePr
 
 	return conclude(_stream, _linePrefix, _formatted);
 }
+
+void SMTCheckerTest::filterObtainedErrors()
+{
+	SyntaxTest::filterObtainedErrors();
+
+	static auto removeCex = [](vector<SyntaxTestError>& errors) {
+		for (auto& e: errors)
+			if (
+				auto cexPos = e.message.find("\\nCounterexample");
+				cexPos != string::npos
+			)
+				e.message = string(e.message.begin(), e.message.begin() + static_cast<long int>(cexPos));
+	};
+
+	if (m_ignoreCex)
+	{
+		removeCex(m_expectations);
+		removeCex(m_errorList);
+	}
```

Comment:
```suggestion
		removeCex(m_errorList);
```

We can assume no counterexample in the expectations in these cases, can't we?

Reply:
That's the thing, I would rather allow the test file to have expectations which are generated by whoever wrote the PR locally, and we just ignore them in the test.